tcms/rpc/tests/test_testexecutionstatus.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=attribute-defined-outside-init, invalid-name, objects-update-used
+
+from xmlrpc.client import ProtocolError
+from tcms.rpc.tests.utils import APITestCase, APIPermissionsTestCase
+
+
+class TestFilter(APITestCase):
+    """Test TestExecutionStatus.filter"""
+
+    def test_filter_statuses(self):
+        execution_status_weights = self.rpc_client.TestExecutionStatus.filter(
+            {"weight__lt": 0}
+        )
+        self.assertGreater(len(execution_status_weights), 0)
```

Comment:
As-is this scenario is identical with the permission scenario below.  Below we're interested whether or not the permission decorator works and thus a non-empty result is sufficient to validate that. Here we're interested whether or not functionality of this API method is as expected. 

So the result should be non-empty but the result is a list of dict objects whose keys match model fields, e.g. name, weight, color. It is important to check that these fields are available otherwise the API isn't of much use to the caller. Also since you are filtering by weight all of the returned results should have a negative weight.

Reply:
got it, making changes accordingly.